src/css/search.css

```diff
@@ -0,0 +1,58 @@
+/* Search Bar */
+.ui.fluid.dropdown {
+  background-color: #000;
+  border: 1px #222 solid;
+  min-height: unset;
+}
+
+.ui.fluid.dropdown > .dropdown.icon {
+  color: #222;
+}
+
+.ui.multiple.search.dropdown > input.search {
+  color: #fff;
+}
+
+/* Selection Label */
+.ui.multiple.dropdown > .label {
+  background-color: #222;
+  color: #fff;
+  vertical-align: middle;
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
+
+i.icon.delete {
+  margin-left: 0.25em;
+  border-left: 0.1em rgba(255, 255, 255, 0.1) solid;
+}
+
+i.icon.delete:before {
+  content: "X";
+  color: #5c5c5c;
+  font-style: normal;
+  padding-left: 0.25em;
+}
+
+/* Dropdown */
+.ui.selection.active.dropdown .menu {
```

Comment:
This also needs a `border-color` declaration — the imported style makes it blue, which we don't want. Add `border-color: #222` to this style and add `.ui.selection.active.dropdown:hover .menu` to this line just to cover our bases.

Reply:
Done - do we want a lighter color to indicate that the element is focused? Can do as a follow up.

Reply:
Went ahead and made the hover color a lighter shade of gray